### PR TITLE
:mag_right:

### PR DESCRIPTION
**Pull request by contolini**

### CODE REVIEW

**Comment by contolini** [comment]
Ping @chosak

File: scripts/search-cfgov.coffee [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        dd = today.getDate()
+        mm = today.getMonth() + 1
+        yyyy = today.getFullYear()
+        today = mm + '/' + dd + '/' + yyyy
+        desc = "Consumerfinance.gov search results for #{query} on #{today} by #{msg.envelope.user.name}"
+
+        if result.length < 10
+          msg.send "No results found :("
+        else
+          gist {content: result, description: desc, fileExtension: 'md'}, (err, resp, data) ->
```
Comment (contolini) [inline scripts/search-cfgov.coffee:55]: We can replace `'md'` with `'csv'` to create a CSV file.

File: scripts/search-cfgov.coffee [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      result = null
+      try
+        result = body
+        today = new Date
+        dd = today.getDate()
+        mm = today.getMonth() + 1
+        yyyy = today.getFullYear()
+        today = mm + '/' + dd + '/' + yyyy
+        desc = "Consumerfinance.gov search results for #{query} on #{today} by #{msg.envelope.user.name}"
```
Comment (chosak) [inline scripts/search-cfgov.coffee:50]: Including the current date here feels less useful than including the date of the crawl. This is a good first implementation, but what do you think about modifying the crawler webservice to return JSON with both the matches and the timestamp of the last commit?

**Review by chosak — APPROVED** [review]

File: scripts/search-cfgov.coffee (hunk 2 above)
Comment (contolini) [reply scripts/search-cfgov.coffee:50]: Brilliant. 👍

**Comment by contolini** [comment]
@ascott1 please add me as an admin to this repo when you have a sec

**Pull request merged**